remme/clients/basic.py

```diff
@@ -47,10 +47,17 @@
 from remme.protos.transaction_pb2 import TransactionPayload
 from remme.shared.exceptions import ClientException, KeyNotFound
 from remme.shared.utils import hash512, get_batch_id, message_to_dict
+
+from remme.shared.exceptions import ClientException
+from remme.shared.exceptions import KeyNotFound
+from remme.shared.utils import hash512
+
 from remme.shared.stream import Stream
-from remme.tp.account import AccountHandler, is_address
+from remme.tp.account import *
```

Comment:
Why implicit import?

Reply:
cyclic imports

Reply:
which is bad and may arise in the future, but allows shared code for transfers

Reply:
```
In [1]: import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
**Explicit is better than implicit.**
Simple is better than complex.
Complex is better than complicated.
Flat is better than nested.
Sparse is better than dense.
Readability counts.
Special cases aren't special enough to break the rules.
Although practicality beats purity.
Errors should never pass silently.
Unless explicitly silenced.
In the face of ambiguity, refuse the temptation to guess.
There should be one-- and preferably only one --obvious way to do it.
Although that way may not be obvious at first unless you're Dutch.
Now is better than never.
Although never is often better than *right* now.
If the implementation is hard to explain, it's a bad idea.
If the implementation is easy to explain, it may be a good idea.
Namespaces are one honking great idea -- let's do more of those!
```
It is the first point. Second, If cycled import will have occurred (possible in the future), imports used inside in function. We do not need redundant imports of some funcs and decrease of readability.